phpcs.xml

```diff
@@ -3,6 +3,7 @@
   <description>Created with the PHP Coding Standard Generator. http://edorian.github.com/php-coding-standard-generator/</description>
   <exclude-pattern>./static</exclude-pattern>
   <exclude-pattern>./vendor</exclude-pattern>
+  <exclude-pattern>./templates</exclude-pattern>
```

Comment:
Why not have it beautified as well?

Reply:
This violated the linter: `<?php if (isset($stacktrace)) : ?>` because it wants it to be `<?php if (isset($stacktrace)) : ?>. O could not be bothered to fix the phpcs. The linter is mostly for bridge authors and PRs.

Reply:
The issue is that this style is better: `<?php if (isset($stacktrace)): ?>` but is not allowed by the linter.